Gemfile

```diff
@@ -1,4 +1,12 @@
-source 'https://rubygems.org'
+source "https://rubygems.org"
 
-# Specify your gem's dependencies in octopress-paginate.gemspec
-gemspec
+gemspec :name => "octopress-paginate"
+
+gem "rake", "~> 11.0"
+
+group :development do
+  gem "clash"
```

Comment:
Are these being installed? The error I see tells me that this isn't being installed...